src/CaptureClient/CaptureSaveToFileProcessor.cpp

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "CaptureClient/CaptureEventProcessor.h"
+#include "CaptureFile/CaptureFile.h"
+#include "CaptureFile/CaptureFileOutputStream.h"
+#include "OrbitBase/MakeUniqueForOverwrite.h"
+#include "capture_data.pb.h"
+
+using orbit_capture_file::CaptureFile;
+using orbit_capture_file::CaptureFileOutputStream;
+using orbit_client_protos::UserDefinedCaptureInfo;
+using orbit_grpc_protos::ClientCaptureEvent;
+
+namespace orbit_capture_client {
+
+namespace {
+
+class CaptureSaveToFileProcessor : public CaptureEventProcessor {
```

Comment:
I don't really like this name, I think it's weird to have "SaveToFile" in the middle. I would go for something like `SaveToFileCaptureEventProcessor`.